lib/zendesk_apps_support/package.rb

```diff
@@ -263,6 +272,10 @@ def has_valid_manifest?(errors)
       has_manifest? && errors.flatten.empty?
     end
 
+    def requirements_marketing_iframe_only_app?(manifest)
+      manifest.requirements_only? || manifest.marketing_only? || manifest.iframe_only?
+    end
```

Comment:
👍

Reply:
Personally I find this a bit unnecessary but included it as otherwise Rubocop complained the complexity of the `Package` class was too high. Good suggestion on making these private methods

Reply:
👍

Reply:
The name of this method seems to imply the app is all of those things at the same time.